elements/pfe-icon-panel/src/pfe-icon-panel.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Icon Panel",
+  "description": "This element creates a header, body, and footer region in which to place content or other components.",
+  "type": "object",
+  "tag": "pfe-icon-panel",
+  "class": "pfe-icon-panel",
+  "category": "combo",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "header": {
+          "title": "Header",
+          "type": "array",
+          "namedSlot": true,
+          "items": {
+            "title": "Header item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        },
+        "body": {
+          "title": "Body",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "title": "Body item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        },
+        "footer": {
+          "title": "Footer",
+          "type": "array",
+          "namedSlot": true,
+          "maxItems": 3,
+          "items": {
+            "title": "Footer item",
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "icon": {
+          "title": "Icon",
+          "type": "string"
```

Comment:
Can you add `"prefixed": false` to these attributes so it's clear they're unprefixed?